src/generators/android/templates/class.java.hbs

```diff
@@ -0,0 +1,59 @@
+/**
+* {{autogeneratedFileWarning}}
+*/
+package com.segment.generated;
+
+import java.util.*;
+import com.segment.analytics.Properties;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public final class {{name}} extends SEGSerializable {
+  private Properties properties;
+
+  {{propertiesGetterSetter name}}
+
+  /**
+  * Builder for {@link {{name ~}} }
+  */
+  public static class Builder {
+    private Properties properties;
+
+    /**
+    * Builder for {@link {{name ~}} }
+    */
+    public Builder() {
+      properties = new Properties();
+    }
+
+  {{#each properties}}
+
+    /**
+     {{#if description}}
+     * {{description}}
+     {{/if}}
+     {{#if isRequired}}
+     * This property is required to generate a valid {{../name}} object
+     {{/if}}
+     */
+    {{builderFunctionSignature name modifiers type}} {
+    {{builderFunctionBody name rawName type}}
+    } 
+    
+  {{/each}}
+
+    /**
+    * Build an instance of {@link {{name ~}} }
+    */
+    public {{name}} build() {
+  {{#each properties ~}}
```

Comment:
Do we need the `~` here? Or does this help with formatting?

Reply:
Yeah the tilde is for controlling whitespace - https://handlebarsjs.com/guide/expressions.html#whitespace-control

I'll see if we can remove it where it's being used.